packages/nuxt/src/app/composables/state.ts

```diff
@@ -35,3 +36,21 @@ export function useState <T> (...args: any): Ref<T> {
   }
   return state
 }
+
+export function clearNuxtState (
+  keys?: string | string[] | ((key: string) => boolean)
+): void {
+  const nuxtApp = useNuxtApp()
+  const _allKeys = Object.keys(nuxtApp.payload.state)
+  const _keys: string[] = !keys
+    ? _allKeys
+    : typeof keys === 'function'
+      ? _allKeys.filter(keys)
+      : Array.isArray(keys) ? keys : [keys]
+
+  for (const key of _keys) {
+    if (key in nuxtApp.payload.state) {
```

Comment:
keys in `_keys` are not started with `useStateKeyPrefix("$s")`. Should the if statement be `("$s" + key in nuxtApp.payload.state)`?

Reply:
quite right!